package io.github.curso.msavaliadorcredito.exception;

public class DadosClienteNotFoundException extends Exception {
    public DadosClienteNotFoundException() {
        super("Dados do cliente n√£o encontrados para o CPF informado.");
    }
}
